packages/smart-wallet/src/smartWallet.js

```diff
@@ -494,6 +563,154 @@ export const prepareSmartWallet = (baggage, shared) => {
           void helper.watchPurse(purse);
           return purse;
         },
+
+        /**
+         * see https://github.com/Agoric/agoric-sdk/issues/8445 and
+         * https://github.com/Agoric/agoric-sdk/issues/8286. As originally
+         * released, the smartWallet didn't durably monitor the promises for the
+         * outcomes of offers, and would have dropped them on upgrade of Zoe or
+         * the smartWallet itself. Using watchedPromises, (see offerWatcher.js)
+         * we've addressed the problem for new offers. This function will
+         * backfill the solution for offers that were outstanding before the
+         * transition to incarnation 2 of the smartWallet.
+         */
+        async repairUnwatchedSeats() {
+          const { state, facets } = this;
+          const { address, invitationPurse } = state;
+          const { liveOffers, liveOfferSeats } = state;
+          const { zoe, agoricNames } = shared;
+          const { invitationBrand, invitationIssuer } = shared;
```

Comment:
I can't tell whether it was intentional to separate them. Please make these two lines or explain why it's four.

Reply:
I hate prettier.

It used to turn 2 lines here or 4 lines into 10 lines. Now it's happy with two lines per declaration. Hurrah!
But with longer names or more names, it will jump from 2 lines to 6, and I'll go back to multiple declarations, or not declaring them all, or declaring some later.